Docs - Traceloop docs (#8897)
PostHog/posthog.com: contents/docs/ai-engineering/traceloop-posthog.md

---
title: Integrating with Langfuse
availability:
    free: full
    selfServe: full
    enterprise: full
---

> > This integration is currently an [opt-in public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).

Reviewer: ```suggestion
> This integration is currently an [opt-in public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://us.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
```

You can integrate with [Traceloop](https://www.traceloop.com/) and bring data into PostHog for analysis. Additionally, we offer a dashboard template to help you quickly get insights into your LLM product. 

## How to install the integration

1. Sign up for [Traceloop](https://www.traceloop.com/) and add it to your app.

> 2. Go to the [integrations page](https://app.traceloop.com/settings/integrations) in your Traceloop dashboard and click on the Posthog card.

Reviewer: ```suggestion
2. Go to the [integrations page](https://app.traceloop.com/settings/integrations) in your Traceloop dashboard and click on the PostHog card.
```

> ![PostHog in Traceloop settings](https://res.cloudinary.com/dmukukwp6/image/upload/v1720097489/posthog.com/contents/posthog-traceloop.png)

Reviewer: We are their only integration 😅  honored

3. Enter in your PostHog host and project API key (you can find these in your [PostHog project settings](https://us.posthog.com/settings/project)).

> 4. Select the environment you want to connect to Posthog and click **Enable**.

Reviewer: ```suggestion
4. Select the environment you want to connect to PostHog and click **Enable**.
```

Traceloop will now send events to PostHog under the name `traceloop span`.

## Using the Traceloop dashboard template

> Once you've installed the integration, [dashboard templates](/docs/product-analytics/dashboards) help you quickly set up relevant insights. You can see an example [Traceloop dashboard here](https://us.posthog.com/shared/tpX9kUd5BbGkdjxQE8YhCskNuYA7Jw).

Reviewer: ```suggestion
Once you've installed the integration, our Traceloop [dashboard template](/docs/product-analytics/dashboards) helps you quickly set up relevant insights. You can see an [example dashboard here](https://us.posthog.com/shared/tpX9kUd5BbGkdjxQE8YhCskNuYA7Jw).
```
